en-US/firefox-ios.xliff

```diff
@@ -1969,6 +2078,16 @@
         <target>New Tab Settings</target>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.OfferClipboardBar.Status">
+        <source>When opening Firefox</source>
+        <target>When opening Firefox</target>
+        <note>Status text of setting to enable the Go to Copied URL feature - https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349</note>
```

Comment:
While the screenshot is great, an additional explanation that avoids leaving Pontoon while translating is even better. E.g.: `Description displayed under the ”Offer to open copied link” option.`